upgrades/v2/README.md

```diff
@@ -32,3 +32,95 @@ The Session functions to get and save AccountBehaviors were removed as well. The
   AccountBehaviors accountBehaviors = AccountBehaviors.loadFromSession();
   accountBehaviors.saveToSession();
 ```
+
+#### ServiceIdentifier
+
+Session.ServiceIdentifier has been removed. If the code is still using it there are a few options:
+
+`Session.ServiceIdentifier.Session` can be changed to `Scope.Session`.
+
+If other values are used, a determination needs to be made. Is the value written or read from the session store, also written or read by another connector?
+
+*If, no:*
+
+`Session.ServiceIdentifier.??` can be changed to `Scope.Service`
+
+*If, yes:*
+  
+The key needs to remain the same or measures need to be taken to migrate away from the current key.
+
+The way to keep key the same, define a stand-in Scope:
+
+```java
+public enum ArchitectIdentifier implements ScopeKeyGenerator {
+  Architect {
+    @Override
+    public String generate() {
+      return "Architect";
+    }
+  }
+}
+```
+
```

Comment:
I really dig all this, thanks! I know I'll be looking at the readme when I run into this :D